schema-registry/src/main/scala/net.manub.embeddedkafka/schemaregistry/streams/EmbeddedKafkaStreamsWithSchemaRegistry.scala

```diff
@@ -0,0 +1,54 @@
+package net.manub.embeddedkafka.schemaregistry.streams
+
+import net.manub.embeddedkafka.UUIDs
+import net.manub.embeddedkafka.schemaregistry.{
+  EmbeddedKafkaConfigWithSchemaRegistry,
+  EmbeddedKafkaWithSchemaRegistry,
+  _
+}
+import net.manub.embeddedkafka.streams.TestStreamsConfig
+import org.apache.kafka.streams.{KafkaStreams, Topology}
+import org.scalatest.Suite
+
+/** Helper trait for testing Kafka Streams.
+  * It creates an embedded Kafka Instance for each test case.
+  * Use `runStreams` to execute your streams.
+  */
+trait EmbeddedKafkaStreamsWithSchemaRegistry
```

Comment:
Could we drop the "WithSchemaRegistry" suffix since this class is already on a different package?